css/style.css

```diff
@@ -62,13 +62,13 @@ main {
 }
 
 .menu-btn {
```

Comment:
- [ ] Please, kindly move the close button to the upper right corner of the screen, like in Figma (you could use `position: absolute` for this).
- [ ] Please add more separation (padding or margin) between the mobile menu items, to match the Figma design.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/156192322-c191854c-0d24-4293-a7b1-1fcca205abd4.png) | ![image](https://user-images.githubusercontent.com/57421823/156192360-2a2d0298-0ae7-41a4-bf8c-e66a680b49c1.png)

That way, your project will look more like the given design 💪